go/vumitools/routing.py

```diff
@@ -790,10 +797,12 @@ def process_inbound(self, config, msg, connector_name):
         connector_type = self.connector_type(connector_name)
         src_conn = self.acquire_source(msg, connector_type, self.INBOUND)
 
-        if self.billing_inbound_connector:
+        if self.billing_enabled:
             if connector_type == self.TRANSPORT_TAG:
                 yield self.publish_inbound_to_billing(config, msg)
                 return
+
+        if self.billing_inbound_connector:
             if connector_type == self.BILLING:
                 # set the src_conn to the transport and keep routing
                 src_conn = str(GoConnector.for_transport_tag(*msg_mdh.tag))
```

Comment:
Does this need to be wrapped in `if self.billing_inbound_connector`? There isn't any other way to receive a message with that connector type.